.circleci/config.yml

```diff
@@ -260,13 +260,44 @@ jobs:
       - store_artifacts:
           path: dist/checksums.txt
 
+  direct_connect_integration_test:
+    executor: linux
+    steps:
+      - checkout
+      - run: git submodule update --init
+      - go_build
+      - run:
+          name: Start the direct connect tunnel
+          command: |
+            RAINFOREST_API_TOKEN=$RAINFOREST_DC_API_TOKEN ./rainforest --skip-update direct-connect --tunnel-id 6
```

Comment:
I think the `tunnel-id` argument is optional, you can remove it if you want.

Reply:
I wanted to leave it to be explicit, that way if someone tries to swap a token things don't break in weird ways, instead the tunnel will fail to start properly with some authentication error.

Reply:
Makes sense!